products/esa-griml-iml-fv1/collection.json

```
@@ -0,0 +1,120 @@
+    "links": [
+      {
+        "rel": "root",
+        "href": "../../catalog.json",
+        "type": "application/json",
+        "title": "Open Science Catalog"
+      },
+      {
+        "rel": "via",
+        "href": "https://doi.org/10.22008/FK2/MBKW9N",
+        "title": "Access"
+      },
+      {
+        "rel": "via",
+        "href": "https://doi.org/10.1038/s41598-021-83509-1",
+        "title": "Documentation"
+      },
+      {
+        "rel": "parent",
+        "href": "../catalog.json",
+        "type": "application/json",
+        "title": "Products"
+      },
+      {
+        "rel": "related",
+        "href": "../../projects/examining-greenland-s-ice-marginal-lakes-under-a-changing-climate-griml/collection.json",
+        "type": "application/json",
+        "title": "Project: (LPF) GRIML"
+      },
+      {
+        "rel": "related",
+        "href": "../../themes/cryosphere/catalog.json",
+        "type": "application/json",
+        "title": "Theme: Cryosphere"
+      },
+      {
+        "rel": "related",
+        "href": "../../variables/lake-surface-temperature/catalog.json",
+        "type": "application/json",
+        "title": "Variable: Lake Surface Temperature"
+      },
+      {
+        "rel": "related",
+        "href": "../../eo-missions/landsat-8/catalog.json",
+        "type": "application/json",
+        "title": "EO Mission: Landsat-8"
+      },
+      {
+        "rel": "related",
+        "href": "../../eo-missions/landsat-9/catalog.json",
+        "type": "application/json",
+        "title": "EO Mission: Landsat-9"
+      },
+      {
+        "rel": "related",
+        "href": "../../eo-missions/sentinel-1/catalog.json",
+        "type": "application/json",
+        "title": "EO Mission: Sentinel-1"
+      },
+      {
+        "rel": "self",
+        "href": "https://esa-earthcode.github.io/open-science-catalog-metadata/products/esa-griml-iml-fv1/collection.json",
+        "type": "application/json"
+      }
+    ],
+    "stac_extensions": [
+      "https://stac-extensions.github.io/osc/v1.0.0-rc.3/schema.json",
+      "https://stac-extensions.github.io/cf/v0.2.0/schema.json"
+    ],
+    "osc:project": "examining-greenland-s-ice-marginal-lakes-under-a-changing-climate-griml",
+    "osc:status": "completed",
+    "osc:region": "Greenland",
+    "osc:type": "product",
+    "created": "2024-11-08T00:00:00Z",
+    "start_datetime": "2016-01-01T00:00:00Z",
+    "end_datetime": "2023-01-01T23:59:59Z",
+    "osc:themes": [
+      "cryosphere"
+    ],
+    "osc:variables": [
+      "land-surface-temperature"
+    ],
+    "osc:missions": [
+      "landsat-8",
+      "landsat-9",
+      "sentinel-1"
+    ],

```

Add Sentinel-2 here also.

```suggestion
    "sentinel-2"
    ],

```